File: EduChemSuite.API/EduChemSuite.API/Models/SchoolModel.cs
namespace EduChemSuite.API.Models;

public class SchoolModel : BaseModel
{
    public required String Name { get; set; }
    public virtual ICollection<UserModel>? Staff { get; set; }
    public virtual ICollection<UserModel>? Students { get; set; }
    public required Guid DistrictId { get; set; }
    public DistrictModel? District { get; set; }
}

File: EduChemSuite.UI/UI/src/app/_models/SchoolModel.ts
import {BaseModel} from "./BaseModel";
import {UserModel} from "./UserModel";
import {DistrictModel} from "./DistrictModel";

export class SchoolModel extends BaseModel {
  name?:string;
  staff?: UserModel[];
  students?: UserModel[];
  districtId?: string;
  district?: DistrictModel;
}
